### PR TITLE
Refresh the Background Services page and add 2 missing sections

### LINKED ISSUE #1867: Page is missing information about Background Services
Hi, The referenced docs page is missing help on a number of features on the Application tab of the dev tools.

Specifically, Payment Handler, Perioding Background Sync (as opposed to Background Sync) and Reporting API.

image

Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 004be442-ec1f-1231-4b3a-96d66bcd625c
* Version Independent ID: 5fd81298-0533-7825-26d5-c09060fcecd7
* Content: [Debug background services - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/background-services?source=docs)
* Content Source: [microsoft-edge/devtools-guide-chromium/javascript/background-services.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/devtools-guide-chromium/javascript/background-services.md)
* Product: **microsoft-edge**
* Technology: **devtools**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
This PR updates all screenshots on the Background Services docs, and also simplifies/clarifies a bit the intro.
More importantly it adds 2 missing sections: Reporting API and Periodic Background Sync.

There is yet another section in this tool that's still not documented: Payment Handler. I am having trouble using this tool successfully to log payment requests and can therefore not document it at this point.

In the meantime, users can find information about it here: https://developer.chrome.com/blog/new-in-devtools-78/#payments

* **Before**: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/background-services
* **After**: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/background-services?branch=pr-en-us-1876

Fixes #1867.

### CODE REVIEW

**Comment by hxlnt** [comment]
@captainbrosset Thanks--this is great! Regarding the issues you ran into with Payment Handler, was this due to lack of a test page using that API to run against? Or is there a need for us to investigate this as a potential bug?

**Comment by captainbrosset** [comment]
@hxlnt I did find test pages to try the tool out: https://rsolomakhin.github.io/pr/bob/ (and others on https://rsolomakhin.github.io/pr/). But couldn't get events to show up in the tool either in Chrome or Edge. So either there's a bug with the tool, or I'm not understanding how this works well enough.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 1]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
 <!-- ====================================================================== -->
 ## Background Fetch
 
-The **Background Fetch API** enables a **service worker** to reliably download large resources, like movies or podcasts, as a background service.  To log Background Fetch event for 3 days, even when DevTools isn't open:
-
-<!--Todo: add background fetch api section when available -->
+The **Background Fetch API** enables the **service worker** of a Progressive Web App to reliably download large resources, like movies or podcasts, as a background service.  To log Background Fetch events for 3 days, even when DevTools isn't open:
 
-1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:45]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 2]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
 <!-- ====================================================================== -->
 ## Background Sync
 
-The **Background Sync API** enables an offline **service worker** to send data to a server once it has re-established a reliable internet connection.  To log Background Sync events for 3 days, even when DevTools isn't open:
+The **Background Sync API** enables the offline **service worker** of a Progressive Web App to send data to a server once it has re-established a reliable internet connection.  To log Background Sync events for 3 days, even when DevTools isn't open:
 
-<!--Todo: add background sync api section when available -->
-
-1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:68]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 3]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
+   ![View the details of an event in the Background Sync pane.](./images/application-background-sync-details.png)
 
 
 <!-- ====================================================================== -->
 ## Notifications
 
 After a **service worker** has received a [Push Message](https://developer.mozilla.org/docs/Web/API/Push_API) from a server, the service worker uses the [Notifications API](https://developer.mozilla.org/docs/Web/API/Notifications_API) to display the data to a user.  To log Notifications for 3 days, even when DevTools isn't open:
 
-1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:90]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 4]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
+
+
+<!-- ====================================================================== -->
+## Periodic Background Sync
+
+The **Periodic Background Sync API** enables the **service worker** of a Progressive Web App to retrieve data from a server, periodically, even when the website isn't opened. To learn more about the **Periodic Background Sync API**, see [Use the Periodic Background Sync API to regularly get fresh content](/microsoft-edge/progressive-web-apps-chromium/how-to/background-syncs#use-the-periodic-background-sync-api-to-regularly-get-fresh-content).
+
+To log Periodic Background Sync events for 3 days, even when DevTools isn't open:
+
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:114]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 5]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
 
-To display a push notification to a user, a **service worker** must first use the [Push Message API](https://developer.mozilla.org/docs/Web/API/Push_API) to receive data from a server.  When the service worker is ready to display the notification, it uses the [Notifications API](https://developer.mozilla.org/docs/Web/API/Notifications_API).  To log Push Messages for 3 days, even when DevTools isn't open:
+To display a push notification to a user, the **service worker** of a Progressive Web App must first use the [Push Message API](https://developer.mozilla.org/docs/Web/API/Push_API) to receive data from a server.  When the service worker is ready to display the notification, it uses the [Notifications API](https://developer.mozilla.org/docs/Web/API/Notifications_API).  To log Push Messages for 3 days, even when DevTools isn't open:
 
-1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:132]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md [hunk 6]
```diff
@@ -22,111 +22,156 @@ ms.date: 05/04/2021
...
+
+
+<!-- ====================================================================== -->
+## Reporting API
+
+The **Reporting API** enables web developers to receive reports of security violations, deprecated API calls, and others, from their production websites.
+
+To see reports that have been sent by the browser using the **Reporting API**:
+
+1. Open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/background-services.md:156]:
```suggestion
1. Open DevTools by right-clicking in the webpage and selecting **Inspect**.  Or by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
```

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 3 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 5 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 6 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 2 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

File: microsoft-edge/devtools-guide-chromium/javascript/background-services.md (hunk 4 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/background-services.md]: Done, and removed `DevTools opens`.

**Pull request merged**